### DISCUSSION TITLE
不是，绳网的评论回复都发邮箱里来了？！

### DISCUSSION THREAD

**Discussion by fashibuding1214** [share121/inter-knot#3060] [其他]
![IMG_20240819_152801](https://github.com/user-attachments/assets/bd893bce-938b-4bc4-8bec-e8c16b2a3a36)
为了上绳网特地注册了账号，此前没用过
发现在一个帖子下回复评论后，后续其他绳匠在帖子下的评论会被发到自己的邮箱里来？？自己发的贴的别人的回复也是，这样感觉自己的邮箱会被塞满，各位精英绳匠有这样的情况吗，怎么解决<img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" >

**Comment by LT-UT** [comment]
代码就是这么写的，可以去找绳网作者反映
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by lllliufeng** [comment]
可以在github里关掉哦(´・ω・`)

**Comment by Nie7bai** [comment]
绳网里有相关的帖子，可以搜索一下

**Comment by youyiMC** [comment]
那就注册一个Outlook邮箱，反正不要手机号，就专门用来接绳网消息（我就这么干的）<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/62158b26331a045bbaabf48d1fc5c8eb_980401343708652819.png" alt="阿姬灵感" width="75" >

**Comment by Goodname-LOL** [comment]
点这里可以调整的哦
![image](https://github.com/user-attachments/assets/abd63a4d-96bb-4eac-8631-d54dfb1aa7e4)

**Reply by Goodname-LOL** [reply]
右下角画红圈那个

**Reply by LT-UT** [reply]
谢谢！
<img src="https://img-static.mihoyo.com/communityweb/upload/d41b52daa44ef813710cf4b482a256b2.png" alt="三月七_点赞" width="75" >

**Comment by kawasaki2333** [comment]
这个是github订阅，绳网是基于github弄出来的平台，你找绳网作者没用，得自己会去取消订阅

**Comment by cleo66** [comment]
天呐，看见自己了，诶嘿<img src="https://bbs-static.miyoushe.com/static/2023/04/28/d7dd7cd39d84b9964a13f157f09ed7a0_4748799527738820824.png" alt="米仔_探头" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/04/28/94e8741c7466beef76c053050f8fe19e_75814081549363228.png" alt="米仔_贴贴" width="75" >
